src/components/formatters/test/meta_formatter_test.cc

```diff
@@ -68,20 +68,6 @@ TEST_F(CMetaFormatterTestHelper,
       CMetaFormatter::CreateObjectByPattern(object2, schema, result_object2);
   EXPECT_TRUE(creationresult);
 
-  // Uncomment code to print objects in console
```

Comment:
@AByzhynar 
I think task was to take only RC changes but this does not have to do with these changes